docs/content/faq.md

```diff
@@ -144,5 +144,39 @@ The Azure mixins are going to be full fledged tools that communicate with the
 Azure APIs, and provide an improved user experience. It all depends on how much
 time you want to invest, and what you are starting from.
 
+## Does the Porter has its own Registry like DockerHub, ACR,Quay etc?
+
+ No, Porter does have its Registry, rather use an OCI compliant artifact store. And now DockerHub is also OCI compliant registry. The list of registries that works with CNAB ie bundles [here](https://porter.sh/compatible-registries/) .
```

Comment:
I realize that this is most likely a copy/paste of my own words from Slack, but I now realize that it is unclear. Let's update it to the following to avoid confusion:

```suggestion
 No, Porter does not have a dedicated registry for hosting bundles. You can use any OCI registry, such as DockerHub, with Porter. We have compiled a [list of compatible registries](https://porter.sh/compatible-registries/) that have been tested with Porter.
```